test-suite/src/components/RequestForm.tsx

```diff
@@ -60,6 +61,47 @@ const RequestForm = ({ name, specs, template, globals }: Props) => {
     return errorMessage;
   };
 
+  const textInputElement = (field: TestRequestField, key: keyof AdaptorRequest) => (
```

Comment:
I would agree with this approach if we were using different input components, such as one being <Date and one being <Input, but as they are the same component with only the 'placeholder' prop being the differentiator, it might be better to extend the schema. 

Similar to `date?` we could also have `placeholder?: string` and use `field.placeholder` as the placeholder prop. Which keeps the input component generic. 

This way if calling that value returns undefined, the component won't render with a placeholder prop. And if we have more inputs that we want a placeholder, we don't have to duplicate more input components